docs/templates/signature.md

```diff
@@ -0,0 +1,160 @@
+# Signature
+
+A Model Signature describes the inputs and outputs for a model. It is used to unambiguously define properties of the inputs and outputs, for example if the model inputs a tensor, it can define the shape and datatype of the tensor. 
```

Comment:
```suggestion
A Model Signature describes the inputs and outputs for a model. It is used to unambiguously define properties of the inputs and outputs. For example, if the model inputs is a tensor, it can define the shape and datatype of the tensor. 
```

Reply:
I rephrased the sentence.